extensions/layout/src/org/sbml/jsbml/ext/layout/CubicBezier.java

```diff
@@ -278,8 +284,12 @@ public Point removeBasePoint1() {
   }
 
   /**
+   * Removes the basepoint 2 from this {@link CubicBezier} 
+   * <br/>
```

Comment:
Instead of using <br/>, try to just let an empty line and then put the next section of code between <p>...</p>
I think I had problem in maven with <br/>.